docker/1.15/Dockerfile.eia

```diff
@@ -0,0 +1,119 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+# Specify LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+# Add arguments to achieve the version, python and url
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG HEALTH_CHECK_VERSION=1.6.2
+ARG S3_TF_EI_VERSION=1-5
+ARG S3_TF_VERSION=1-15-2
```

Comment:
Can we use TFS_SHORT_VERSION only, and set the S3_TF_VERSION in the url to be "1-15-0" rather than "1-15-2"?

Reply:
We have been naming our TFS binaries against the TF version inbuilt.. Any changes to that will require automation changes.

Reply:
Ah, okay. Cool. Thanks for clarifying.